extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -201,6 +207,28 @@ class AmpStickyAd extends AMP.BaseElement {
       this.viewport_.updatePaddingBottom(0);
     });
   }
+
+  /**
+   * To check for background-color alpha and force it to be 1.
+   * Whoever call this needs to make sure it's in a vsync.
+   * @private
+   */
+  forceOpacity_() {
+    if (!isExperimentOn(this.win, TAG)) {
+      return;
+    }
+    const background = this.win.getComputedStyle(this.element)
```

Comment:
move this logic to `style.js` with some unit tests. Maybe something like `removeAlphaFromColor(cssColorValue)`